src/userLayout/payment/Checkout.tsx

```diff
@@ -0,0 +1,163 @@
+      // Make sure to disable form submission until Stripe.js has loaded.
+      return;
+    }
+    const cardElement = elements?.getElement(CardElement);
+    console.log(cardElement);
+    if (!cardElement) {
+      console.error("Card element not found");
+      return;
+    }
+    const { error, token } = await stripe.createToken(cardElement);
+    console.log(token);
+    const tokenId = token?.id;
+
+    const payBooking = async (id: string) => {
+      await axios
+        .post(
+          `${baseUrl}/v0/portal/booking/${id}/pay`,
+          { token: tokenId },
+          {
+            headers: {
+              Authorization: `Bearer ${loginData}`,
+            },
+          }
+        )
+        .then((response) => {
+          console.log(response);
+
+          toast.success("Booking paid successfully");
+          navigate("/");
+        })
+        .catch((error) => {
+          if (axios.isAxiosError(error) && error.response) {
+            toast.error(error.response.data.message);
+          }
+          console.log(error);
+        });
+    };
+    payBooking("65a9968ea5d9953dd42d11aa");
+    if (error) {
+      // Show error to your customer (for example, payment details incomplete)
+      console.log(error.message);
+    }
+  };
+  //  get booking details =>
+    useEffect(() => {
+      const getBookingDetails = async () => {
+       
+        try {
+          const response = await axios.get(
+            `${baseUrl}/v0/portal/booking/${id}`,
+            {
+              headers: {
+                Authorization: `Bearer ${loginData}`,
+              },
+            }
+          );
+          setBookingDetails(response);
+          
+        } catch (error) {
+          console.error("Error fetching booking details:");
+       
+        }
+      };
+  
+      if (id) {
+        getBookingDetails();
+      }
+    }, [id]);
+
+  const authContext = useContext(AuthContext);
+  if (!authContext) {
+    // Handle the case where AuthContext is null
+    return null;
+  }
+  const { baseUrl, loginData } = authContext;
+
+  return (
+    <>
+      <Typography sx={{ textAlign: "center" }} variant="h3" color="initial">
+        Payment
+      </Typography>
+      <Typography
+        sx={{ textAlign: "center", color: "gray" }}
+        variant="h6"
+        color="initial"
+      >
+        Kindly follow the instructions below
+      </Typography>
+
+      <Box sx={{ width: "100%", display: "flex", justifyContent: "center" , alignItems:"center" }}>
+        <Grid
+          container
+          sx={{
+            justifyContent: "center",
+            my: "10rem",
+            width: "100%",
+            textAlign: "center",
+          }}
+          lg={12}
+        >
+          <Grid  item xs={10} sm={10} md={10}  lg={5}>
+            <Typography variant="h4" color="initial">
+              Transfer Pembayaran:
+            </Typography>
+            <Typography sx={{mt :"1rem",color :'gray'}}  variant="h4" color="initial">
```

Comment:
As I mentioned before, we should stick to the color for the MUI theme and avoid using fixed colors.